core/lib/dal/src/blocks_dal.rs

```diff
@@ -183,7 +184,8 @@ impl BlocksDal<'_, '_> {
                 l2_l1_compressed_messages, l2_l1_merkle_root, l1_gas_price, l2_fair_gas_price, \
                 rollup_last_leaf_index, zkporter_is_available, bootloader_code_hash, \
                 default_aa_code_hash, base_fee_per_gas, aux_data_hash, pass_through_data_hash, \
-                meta_parameters_hash, protocol_version, \
+                meta_parameters_hash, protocol_version, system_logs, compressed_state_diffs, skip_proof, 
+                gas_per_pubdata_byte_in_block, gas_per_pubdata_limit, \
```

Comment:
No need to select `skip_proof, gas_per_pubdata_byte_in_block, gas_per_pubdata_limit`. Please fix it for all touched queries